examples/read_nodes_serde.rs

```diff
@@ -0,0 +1,81 @@
+// note: to use serde, the feature needs to be enabled
+// run example with:
+//    cargo run --example read_nodes_serde --features="serialize"
+
+use quick_xml::de::from_str;
+use serde::Deserialize;
+
+#[derive(Debug, PartialEq, Default, Deserialize)]
+#[serde(default)]
+struct Translation {
+    #[serde(rename = "@Tag")]
+    tag: String,
+    #[serde(rename = "@Language")]
+    lang: String,
+    #[serde(rename = "Text")]
+    text: String,
```

Comment:
If this is just an example format (and not some that has a name) I think it would be better to show how to read inner text, using `#[serde(rename = "$text")]`. Then 
```xml
    <Translation Tag="HELLO" Language="ja">
      <Text>こんにちは</Text>
    </Translation>
```
becomes
```xml
    <Translation Tag="HELLO" Language="ja">
      こんにちは
    </Translation>
```

Reply:
nice suggestion.  done!